ansible/roles/atmo-vnc/files/Xresources

```diff
@@ -0,0 +1,3 @@
+xterm*faceName: Ubuntu Mono
+xterm*faceSize: 12
+xterm*renderFont: true
```

Comment:
LG2M -- As long as these keynames (`xterm*...`) are valid to ansible, i say we go ahead and :shipit: